src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerAddAudioRequestDownloadIntTest.java

```diff
@@ -74,11 +73,11 @@ void beforeEach() {
             SOME_COURTROOM,
             LocalDateTime.parse(HEARING_DATETIME)
         );
-        CourtCaseEntity courtCase = hearingEntity.getCourtCase();
-        courtCase.addProsecutor("aProsecutor");
-        courtCase.addDefendant("aDefendant");
-        courtCase.addDefence("aDefence");
-        dartsDatabase.save(courtCase);
+        //        CourtCaseEntity courtCase = hearingEntity.getCourtCase();
+        //        courtCase.addProsecutor("aProsecutor");
+        //        courtCase.addDefendant("aDefendant");
+        //        courtCase.addDefence("aDefence");
+        //        dartsDatabase.save(courtCase);
```

Comment:
Just checking if this commenting is intentional.

Reply:
Yes, they were using a method that was only being used in test which has been deleted.  These tests are fixed in part 2 of re-enabling the audio tests